src/features/areas/components/PublicAreaMap.tsx

```diff
@@ -91,14 +91,15 @@ type PublicAreaMapProps = {
 const PublicAreaMap: FC<PublicAreaMapProps> = ({ area }) => {
   const theme = useTheme();
   const classes = useStyles();
-  const createPlace = useCreatePlace(area.organization.id);
   const places = usePlaces(area.organization.id).data || [];
 
   const [selectedPlace, setSelectedPlace] = useState<ZetkinPlace | null>(null);
   const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
   const [dialogStep, setDialogStep] = useState<'place' | 'log'>('place');
   const [returnToMap, setReturnToMap] = useState(false);
   const [standingStill, setStandingStill] = useState(false);
+  const [showCard, setShowCard] = useState(false);
```

Comment:
I think a better name for this is something like `[isCreating, setIsCreating]` because it describes the state, and not just the visual effect of the state. The state of creating may have several visual effects, e.g. we might hide some parts of the UI and show other parts of the UI.